### ISSUE TITLE
Can't scroll stack on mobile

### ISSUE THREAD

**Issue by JonCup** [nextcloud/deck#383]
### Steps to reproduce
1. Try to scroll stack on chrome on Android
2. Can't use scroll bar, scrolling on page doesn't work
3.

### Expected behaviour
Tell us what should happen

### Actual behaviour
Tell us what happens instead

### Server configuration

**Operating system**:

**Web server:**

**Database:**

**PHP version:**

**Nextcloud version:** (see Nextcloud admin page)

**Where did you install Nextcloud from:**

**Signing status:**

```
Login as admin user into your Nextcloud and access
http://example.com/index.php/settings/integrity/failed
paste the results here.
```

**List of activated apps:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```

**Nextcloud configuration:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder

or

Insert your config.php content here
Make sure to remove all sensitive content such as passwords. (e.g. database password, passwordsalt, secret, smtp password, …)
```

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

### Client configuration
**Browser:**

**Operating system:**

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```

**Comment by juliushaertl** [comment]
What version of Deck are you using? There have been some fixes that should be out soon in 0.3.0.

**Comment by juliushaertl** [comment]
0.3.0 is out, please give it a try and report back if there are still issues with that.
